scripts/deploy

```diff
@@ -32,7 +34,8 @@ declare_kubeconfig
 import_image quay.io/submariner/lighthouse-agent
 import_image quay.io/submariner/lighthouse-coredns
 
-run_subm_clusters install_service_import
+echo ${deploytool}
+echo
```

Comment:
Are these unintentional local debugging changes?

Reply:
Yes it was unintentional and removed it.